source/user-manual/agents/remote-upgrading/create-custom-wpk/generate-wpk-package-manually.rst

```diff
@@ -67,13 +67,15 @@ Delete the files that are no longer needed. This step can be skipped, but the si
 
 .. code-block:: console
 
+  $ rm -rf ./{api,framework}
   $ rm -rf doc wodles/oscap/content/* gen_ossec.sh add_localfiles.sh Jenkinsfile*
   $ rm -rf src/{addagent,analysisd,client-agent,config,error_messages,external/*,headers,logcollector,monitord,os_auth,os_crypto,os_csyslogd,os_dbdos_execd}
-  $ rm -rf src/{os_integrator,os_maild,os_netos_regex,os_xml,os_zlib,remoted,reportd,shared,syscheckd,tests,update,wazuh_db,wazuh_modules}
+  $ rm -rf src/{os_integrator,os_maild,os_netos_regex,os_xml,os_zlib,remoted,reportd,shared,syscheckd,tests,update,wazuh_db}
```

Comment:
1. Found a typo in `os_netos_regex`
2. `tests` folder doesn't exist. We have to remove `unit_tests`
3. `update` folder doesn't exist and doesn't correspond to any existing folder

```suggestion
  $ rm -rf src/{os_integrator,os_maild,os_net,os_regex,os_xml,os_zlib,remoted,reportd,shared,syscheckd,unit_tests,wazuh_db}
```